src/sentry/models/project.py

```diff
@@ -227,8 +227,9 @@ def get_audit_log_data(self):
         }
 
     def get_full_name(self):
-        if self.team.name not in self.name:
-            return '%s %s' % (self.team.name, self.name)
+        team_name = self.teams.values_list('name', flat=True).first()
```

Comment:
we're getting rid of full name in the future right

Reply:
yeah